From foo@baz Tue Apr  9 12:12:43 2002
Date: Thu, 01 Nov 2007 13:31:26 -0700
To: Greg KH <greg@kroah.com>
From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: driver core: remove owner field from struct bus_type

This isn't used by anything in the driver core, and by no one in the 204
different usages of it in the kernel tree.  Remove this field so no one
gets any idea that it is needed to be used.

Cc: Kay Sievers <kay.sievers@vrfy.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 include/linux/device.h |    1 -
 1 file changed, 1 deletion(-)

--- a/include/linux/device.h
+++ b/include/linux/device.h
@@ -51,7 +51,6 @@ extern void bus_remove_file(struct bus_t
 
 struct bus_type {
 	const char		* name;
-	struct module		* owner;
 
 	struct kset		subsys;
 	struct kset		*drivers_kset;
